clients.json

```
@@ -722,6 +722,16 @@
     "authors": ["chakrit"]
   },
 
+  {
+    "name": "Rediska",
+    "language": "C#",
+    "url": "https://github.com/pepelev/Rediska",
+    "description": "Rediska is a Redis client for .NET with a focus on flexibility and extensibility.",
+    "authors": ["pepelev"],
```

This should be your Twitter handle, or just empty if you don't use that network,.

```suggestion
    "authors": [],
```

---

It was not clear for me. So far i don't use Twitter.

---

Sure - I'm aware there something left to be desired with the current process :) Thinking about improving this.